spec/support/request_helpers.rb

```diff
@@ -0,0 +1,9 @@
+module RequestHelpers
+  def json_response(response)
```

Comment:
I think `json` is easier to use

Reply:
Ok, I don't have con arguments, let's change it to be just json